app/src/main/java/com/yml/womensafety/navigationdrawer/TipsForWomenSafetyFragment.kt

```diff
@@ -0,0 +1,67 @@
+package com.yml.womensafety.navigationdrawer
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.View
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.yml.womensafety.TipsForWomenSafetyAdapter
+import com.yml.womensafety.R
+
+class TipsForWomenSafetyFragment : Fragment(R.layout.fragment_tips_for_women_safety) {
+
+    private val tipsTextList = ArrayList<SafetyTips>()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
```

Comment:
Use scope functions to assign value to recycler view. 
ex : recyclerview.apply{
layoutManager = LinearLayoutManager()
adapter = Adapter()
}